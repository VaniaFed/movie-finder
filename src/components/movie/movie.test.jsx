import renderer from 'react-test-renderer';
import { Movie } from './index';

const movie = {
    budget: 44000000,
    genres: ['Crime', 'Drama', 'Mystery', 'Western'],
    id: 273248,
    overview:
        'Bounty hunters seek shelter from a raging blizzard and get caught up in a plot of betrayal and deception.',
    poster_path:
        'https://image.tmdb.org/t/p/w500/fqe8JxDNO8B8QfOGTdjh6sPCdSC.jpg',
    release_date: '2015-12-25',
    revenue: 155760117,
    runtime: 167,
    tagline: 'No one comes up here without a damn good reason.',
    title: 'The Hateful Eight',
    vote_average: 7.6,
    vote_count: 5584
};
describe('<Movie />', () => {
    const tree = renderer.create(<Movie movie={movie} />).toJSON();
    it('should be rendered', () => {
        expect(tree).toMatchSnapshot();
    });
});
