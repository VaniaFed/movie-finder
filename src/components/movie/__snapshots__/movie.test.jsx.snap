// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Movie /> should be rendered 1`] = `
<div
  className="movie"
>
  <img
    alt="movieimage"
    className="movie__image"
    src="https://image.tmdb.org/t/p/w500/fqe8JxDNO8B8QfOGTdjh6sPCdSC.jpg"
  />
  <div
    className="movie__info"
  >
    <h2
      className="movie__title"
      data-text="movie-page__movie-title"
    >
      The Hateful Eight
    </h2>
    <p
      className="movie__genres"
    >
      Crime, Drama, Mystery, Western
    </p>
    <p
      className="movie__date"
    >
      2015-12-25
    </p>
    <p
      className="movie__duration"
    >
      167
      m
    </p>
    <p
      className="movie__description"
    >
      Bounty hunters seek shelter from a raging blizzard and get caught up in a plot of betrayal and deception.
    </p>
  </div>
</div>
`;
